### PR TITLE
More code scanning

### PR DESCRIPTION
**Pull request by vincentmacri**
Adds some code scanning tool GitHub recommended. Might be useful, let's see what it does.

### CODE REVIEW

File: .github/workflows/ossar-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# This workflow integrates a collection of open source static analysis tools
+# with GitHub code scanning. For documentation, or to provide feedback, visit
+# https://github.com/github/ossar-action
+name: OSSAR
+
+on:
+  push:
+  pull_request:
```
Comment (tyxchen) [inline .github/workflows/ossar-analysis.yml:8]: Should these keys be set?

Comment (vincentmacri) [reply .github/workflows/ossar-analysis.yml:8]: This is the default workflow for the tool. I believe this sets it to run on all pushes and PRs.

Comment (tyxchen) [reply .github/workflows/ossar-analysis.yml:8]: Wonder if we should have it for pushes and PRs to master only, as we do for the other security workflows?

Comment (vincentmacri) [reply .github/workflows/ossar-analysis.yml:8]: I'd leave it on the default settings first and change later if needed.

**Review by tyxchen — APPROVED** [review]

**Pull request merged**